.java-version

```diff
@@ -1 +1 @@
-1.6
+1.8
```

Comment:
(I actually use this file - for jenv ( http://www.jenv.be/ ), but if people mind I'm ok to remove it from the repo (and add to my local gitignore)